cinema/urls.py

```diff
@@ -0,0 +1,11 @@
+from django.urls import path
+
+from cinema.views import movie_list, change_movie
+
+urlpatterns = [
+    path("movies/", movie_list, name="movie-list"),
+    path("movies/<int:pk>/", change_movie, name="change-movie"),
```

Comment:
maybe it's not important, but you also can read movie in this url, so it's better to write movie_detail